### ISSUE TITLE
[Tuning] Okta.Support.Reset - Wrong path for `userAgent` condition

### ISSUE THREAD

**Issue by PeterJDavid** [panther-labs/panther-analysis#2090]
### Rule ID

Okta.Support.Reset

### Description

Wrong path for `userAgent` condition

**Function:** `rule()`

**Current code:**
```python
return (
    event.deep_get("actor", "alternateId") == "system@okta.com"
    and event.deep_get("transaction", "id") == "unknown"
    and event.deep_get("userAgent", "rawUserAgent") is None   # <— wrong path
    and event.deep_get("client", "geographicalContext", "country") is None
)
```

**Issue:** Okta SystemLog events nest `userAgent` under `client`, not at the top level. There is no top-level `userAgent` field in the schema. The condition `event.deep_get("userAgent", "rawUserAgent") is None` therefore always returns True regardless of the actual `client.userAgent.rawUserAgent` value, making the check an effective no-op.

This is the same pattern category as the singular-vs-nested path bugs we reported in the Slack pack — silent field-path mismatch.

Verified by `DESCRIBE TABLE panther_logs.okta_systemlog` — no top-level `userAgent` column. Sample events with `actor.alternateId == 'system@okta.com'` show:

```json
{
  "client": {
    "userAgent": {
      "browser": "CHROME",
      "os": "Mac OS 15.7.5 (Sequoia)",
      "rawUserAgent": "Mozilla/5.0 ..."
    },
    "ipAddress": "...",
    "geographicalContext": { "country": "United States", ... }
  }
}
```

…demonstrating the correct path is `client.userAgent.rawUserAgent`. The sibling condition in the same function (`event.deep_get("client", "geographicalContext", "country")`) already uses the correct `client.*` nesting, so this looks like an inconsistency / typo rather than intentional.

**Suggested fix:** `event.deep_get("userAgent", "rawUserAgent")` → `event.deep_get("client", "userAgent", "rawUserAgent")`.

### Example Log (redacted)

{
  "eventType": "user.account.reset_password",
  "actor": {
    "alternateId": "system@okta.com",
    "displayName": "Okta System",
    "id": "spr00000000000000000",
    "type": "SystemPrincipal"
  },
  "target": [
    {
      "alternateId": "impacted-user@example.com",
      "displayName": "Impacted User",
      "id": "00u0impacted00000000",
      "type": "User"
    }
  ],
  "client": {
    "device": "Mobile",
    "geographicalContext": {
      "city": "Anywhere",
      "country": "United States",
      "state": "Example"
    },
    "ipAddress": "198.51.100.20",
    "userAgent": {
      "browser": "SAFARI",
      "os": "Mac OS X (iPhone)",
      "rawUserAgent": "Mozilla/5.0 (iPhone; CPU iPhone OS 18_7 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/26.4 Mobile/15E148 Safari/604.1"
    },
    "zone": "null"
  },
  "transaction": {
    "detail": {},
    "id": "00000000000000000000000000000000",
    "type": "WEB"
  },
  "outcome": {
    "reason": "Issued recovery token",
    "result": "SUCCESS"
  }
}
